pkg/minikube/drivers/hyperkit/network_test.go

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hyperkit
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"k8s.io/minikube/pkg/minikube/tests"
+)
+
+var validLeases = []byte(`{
+	name=foo
+	ip_address=1.2.3.4
+	hw_address=1,a1:b2:c3:d4:e5:f6
+	identifier=1,a2:b3:c4:d5:e6:f7
+	lease=0x597e1267
+}
+{
+	name=bar
+	ip_address=192.168.64.3
+	hw_address=1,a4:b5:c6:d7:e8:f9
+	identifier=1,a0:b0:c0:d0:e0:f0
+	lease=0x597e1267
+}`)
+
+func Test_getIpAddressFromFile(t *testing.T) {
```

Comment:
Added a third entry. The code looks by MAC, so I can't really add a test for that case since name isn't part of the API at all.